software/opik.yml

```diff
@@ -0,0 +1,12 @@
+name: "Opik"
+website_url: "https://www.comet.com/site/products/opik/"
+source_code_url: "https://github.com/comet-ml/opik"
+description: "Evaluate, test, and ship LLM applications with a suite of observability tools to calibrate language model outputs across your dev and production lifecycle."
+licenses:
+  - Apache-2.0
+platforms:
+  - Python
+tags:
+  - Monitoring
+  - Self-hosting Solutions
+  - Software Development
```

Comment:
```suggestion
tags:
  - Software Development - API Management
depends_3rdparty: true
```

Reply:
Thanks for the prompt feedback @Ki-er! 

You can find a full list of **Opik releases** (as recent as last week) here: https://github.com/comet-ml/opik/releases
There is also a **changelog** here: https://www.comet.com/docs/opik/changelog/

Let me know if you'd like any additional info!

Reply:
@anmorgan24 what @Ki-er means is that the project has not seen any new releases since September 2023. I also think we should wait for a new release before approving this. Are there plans for a new release?